pingpong/server.py

```
@@ -929,6 +929,377 @@ async def auth_canvas(request: StateRequest):
+            )
+            return jsonrpc_response(
+                {"content": [{"type": "text", "text": result_text}]}
+            )
+        except PanoptoException as e:
+            return jsonrpc_response(
+                {
+                    "content": [{"type": "text", "text": f"Error: {e.detail}"}],
+                    "isError": True,
+                }
+            )
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)